README.md

```diff
@@ -16,6 +16,7 @@ The IAM role for the broker must include at least the following policy:
                 "s3:CreateBucket",
                 "s3:DeleteBucket",
                 "s3:PutBucketPolicy",
+                "s3:PutBucketPublicAccessBlock",
```

Comment:
Don't you need a DeletePublicAccessBlock here too?

Reply:
No because the delete is handled by put https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeletePublicAccessBlock.html